presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -115,11 +115,21 @@ public void deployDefaultConfig(Path configDir, Path dataDir, Path pluginDir)
                 // TODO: Include node.id in install requests
                 updateProperty(configDir.resolve("node.properties"),
                         "node.id", randomUUID().toString());
+                /**
+                 * To prevent conflicts between Presto-Manager configs and
+                 * Presto configs present in {@link #defaultConfig} directory
+                 */
+                Path nodePropertiesPath = configDir.toAbsolutePath().resolve("node.properties");
```

Comment:
is it possible for presto-manager to get this info from the provided configs instead of as a separate property?  because it would be much better for there to be a single source of truth.